source/platforms/github/_tests/_customGitHubRequire.test.ts

```diff
@@ -31,6 +31,12 @@ describe("shouldUseGitHubOverride", () => {
     const parent: any = { filename: dangerPrefix + "./my-import" }
     expect(shouldUseGitHubOverride(module, parent)).toBeTruthy()
   })
+
+  it("ignores modules without a parent", () => {
+    const module = "./peril"
+    const parent: any = undefined
```

Comment:
I could avoid the type here - the following would work as well:
```suggestion
    const parent = undefined
```
but I used what's more consistent with the other test cases.

Let me know if you'd like to drop this `any`.

Reply:
This is fine :+